src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/SubaccountSupervisor.kt

```diff
@@ -432,6 +434,82 @@ internal class SubaccountSupervisor(
         }
     }
 
+    /**
+     * @description Get the childSubaccount number that is available for the given marketId
+     * @param marketId
+     */
+    @Throws(Exception::class)
```

Comment:
`@Throws` here isn't super necessary - it's mostly for Java interop which we don't really care about. (As you can see from your call-site, Kotlin isn't forcing you to handle the exception)